### ISSUE TITLE
Warn When Empty Alt

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#377]
We should add a scan that looks for when `alt=""`. When that occurs, the scan should output a message with the message type "warning" and message body "Alt text is missing. Alt text is required on images that are not decorations."

**Comment by azdak** [comment]
Hey so after a little research, this is actually a tetch thorny:
- Technically empty alt tags are WCAG compliant, and indicate a decorative or non-functional image/element (see [here](https://github.com/dequelabs/axe-core/issues/590)). So this rule would be outside of WCAG standards.
- I can write a custom rule and check for empty tags and feed it into axe, ~but axe only returns node results as `type:` `violation` or `incomplete`- there's no type for "warning."~ (edit: actually to clarify, the `type` is just what axe assigns as the result of the rule test, "pass", "violation", "incomplete", or "error." So this would need to just be a `violation`- is that okay?)
- Similarly, I can set the violation `impact` as one of ['minor', 'moderate', 'serious', 'critical'] ([see here](https://github.com/dequelabs/axe-core/blob/fff9b9da4f4f14071c42a8476e75a63be23d76eb/lib/core/constants.js#L34))- is there a preference?

Just wanted to make sure you want me to go ahead on this considering the above provisos. Also, do you have a test page or two handy with empty alt tags for testing?

**Comment by bbertucc** [comment]
Thanks @azdak. Here are some answers for you:
- I would tag it "incomplete". Axe [treats incomplete as "needs review" (link)](https://docs.deque.com/devtools-for-web/4/en/needs-review-incomplete) - this And that actually raises another issue I found: #378 
- Moderate `impact`
Please go ahead!

Tests:
- https://wordpress.org should return it on the brush stroke with the alt `alt`
- https://www.w3.org/WAI/tutorials/images/decorative/ should return various messages that have `alt=""`

**Comment by azdak** [comment]
V1/testing version of this up on `dev`

**Comment by bbertucc** [comment]
Great. Putting this on me to test.

**Comment by bbertucc** [comment]
This looks great @azdak!
